locales/zh-TW.json

```diff
@@ -149,6 +149,16 @@
 	"Only allow secure html tags (recommended)": "只允許安全的 HTML 標籤 (建議)",
 	"Allow styles": "允許樣式",
 	"Allow dangerous html tags": "允許危險的 HTML 標籤",
-	"Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown.": "Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown.",
-	"⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠": "⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠"
+	"Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown.": "將文本箭頭轉換為完整符號。 ⚠ 注意這或會影響 Markdown 的 HTML 注釋。",
+  "⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠": "⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠",
+  "Default New Note":"預設新筆記類型",
+  "Show only related tags": "只顯示相關標籤",
+  "Snippet Default Language": "程式碼片段預設語言",
+  "Disable Direct Write (It will be applied after restarting)": "停用直接編輯 (重啟後生效)",
```

Comment:
Should we be consistent with `"⚠️ Please restart boostnote after you change the keymap"`? This could be another issue.